README.md

```diff
@@ -130,6 +132,46 @@ import { DeviceResponse } from "@auth0/mdl";
 })();
 ```
 
+### Using an arbitrary session transcript
+
+```js
+import { DeviceResponse } from "@auth0/mdl";
+(() => {
+  let issuerMDoc;
+  let deviceResponseMDoc;
+
+  // This is what the MDL issuer does to generate a credential:
+  {
+    const document = await new Document('org.iso.18013.5.1.mDL')
+      .addIssuerNameSpace('org.iso.18013.5.1', {
+        family_name: 'Jones',
+        given_name: 'Ava',
+        birth_date: '2007-03-25',
+      })
+      .useDigestAlgorithm('SHA-256')
+      .addValidityInfo({
+        signed: new Date(),
+      })
+      .addDeviceKeyInfo({ deviceKey: publicKeyJWK })
+      .sign({
+        issuerPrivateKey,
+        issuerCertificate,
+        alg: 'ES256',
+      });
+    issuerMDoc = new MDoc([document]).encode();
+  }
```

Comment:
minor suggestion: both samples have the same code for `document` and `issuerMDoc` initialisation. Consider something like this:

```js
// This is what the MDL issuer does to generate a credential:
// ...

// This is what the DEVICE does to generate a response using a OID4VP handover:
// ...

// This is what the DEVICE does to generate a response using an arbitrary session transcript:
// ...
```

WDYT?

Reply:
👍  See bfca0c2